cypress/support/commands.ts

```diff
@@ -117,3 +117,15 @@ Cypress.Commands.add("loginSSO", () => {
     });
   });
 });
+
+Cypress.Commands.add("getHelloEmail", () => {
```

Comment:
Can you make this command something like `getEmails` (so it's generic enough for reuse in other tests), and then move all of the chained `.then(...` into the test itself?